sopel/module.py

```diff
@@ -257,6 +255,68 @@ def commands(*command_list):
             # If the command prefix is "\\.", this would trigger on lines
             # starting with either ".j" or ".join".
 
+    You can use a space in the command name to implement subcommands::
+
+        @commands('main sub1', 'main sub2')
+            # For ".main sub1", trigger.group(1) will return "main sub1"
+            # For ".main sub2", trigger.group(1) will return "main sub2"
+            # For ".main", trigger.group(1) will return "main"
+
+    But in that case, be careful with the order of the names: if a more generic
+    pattern is defined first, it will have priority over less generic patterns.
```

Comment:
It'll be hard (if possible at all), because 1) the string has no semantic (we can't infer rules from its meaning) and 2) it can be a regex pattern which we can't sort reliably.

Reply:
Then the doc is fine as-is and I'll mark this note resolved. We'll have a chance to revisit this in 8.x after regex is no longer allowed in command names.